deployment/7_upgrade_1_6.js

```diff
@@ -139,12 +140,12 @@ const deploy = async (network) => {
   // Disable KyberNetwork on the TokenPriceProvider
   // ////////////////////////////////////////////////////
 
-  if (!await TokenPriceProviderWrapper.contract.managers(deploymentWallet.address)) {
-    const TokenPriceProviderAddManagerTx = await TokenPriceProviderWrapper.contract.addManager(deploymentWallet.address);
-    await TokenPriceProviderWrapper.verboseWaitForTransaction(TokenPriceProviderAddManagerTx,
-      `Set ${deploymentWallet.address} as the manager of the TokenPriceProvider`);
-  }
-  const TokenPriceProviderSetKyberNetworkTx = await TokenPriceProviderWrapper.contract.setKyberNetwork("0x0000000000000000000000000000000000000000");
+  const TokenPriceProviderAddManagerTx = await TokenPriceProviderWrapper.contract.addManager(deploymentWallet.address, { gasPrice });
```

Comment:
I noticed that you left out my `if (!await TokenPriceProviderWrapper.contract.managers(deploymentWallet.address)) {` but I think it's ok. Not really needed.

Reply:
I think I dropped it by mistake when doing the rebase. I will add it again.